QA.md

```diff
@@ -1,186 +1,191 @@
-          in project creation, push them to Radicle with `git push rad`, switch
-          to the checkout working directory, do a `git pull`
 
+### How to install? <a href="#user-content-03" id="03">🔗</a>
 
-#### Source browsing
+**On macOS:**
 
-- [ ] Metadata and stats in UI reflect what is in the actual repository; here
-      are a couple commands that will help you get the numbers from
-      a repository:
-  - local branch count
-    `git branch | wc -l`
-  - all unique contributors across all branches
-    `git shortlog --summary --numbered --email --all`
-  - all unique contributors in a specific branch
-    `git shortlog --summary --numbered --email myfunkybranch`
-  - unique commit count across all branches
-    `git rev-list --all --count`
-  - commit count in a specific branch
-    `git rev-list --count myfunkybranch`
-- [ ] `README.md` files are shown by default and markdown is rendered as HTML
-  - [ ] Links to external resources open in external browser
-  - [ ] Links to internal resources don't do anything
-  - [ ] If the project doesn't have a `README.md`, a placeholder is shown
-- [ ] Syntax highlighting works for source files (.toml, .sol, .ts, .svelte)
-- [ ] Binary files show a placeholder
-- [ ] It's possible to navigate to deeper hierarchies via the tree browser
-- [ ] It's possible to select different branches
+  1. open the `radicle-upstream-X.X.X.dmg` package
+  2. install Upstream by dragging the `Radicle Upstream` binary to
+     `/Applications`
 
+**On Linux (AppImage):**
 
-#### Commit browsing
+  1. `chmod +x <PATH_TO_DOWNLOAD>/radicle-upstream-X.X.X.AppImage`
 
-- [ ] Commit tab shows a list of all the commits in the branch that was
-      selected
-- [ ] Clicking on a commit shows the commit metadata as well as the diff
 
+### How to query repo stats from the CLI? <a href="#user-content-04" id="04">🔗</a>
 
-### Misc UI
+```
+  # local branch count
+  git branch | wc -l
 
-- [ ] Clicking on an identifier copies it to the clipboard
+  # all unique contributors across all branches
+  git shortlog --summary --numbered --email --all
 
+  # all unique contributors in a specific branch
+  git shortlog --summary --numbered --email myfunkybranch
 
-#### Global keyboard shortcuts
+  # unique commit count across all branches
+  git rev-list --all --count
 
-- [ ] All documented shortcuts (a list is provided by pressing `?`) work
-- [ ] Only one modal is allowed at a time (no modal stacking possible)
+  # commit count in a specific branch
+  git rev-list --count myfunkybranch
+```
 
-### Proxy Error
 
-- [ ] Killing the proxy process while the app is running shows a blue error
-      screen with the proxy logs.
 
-[re]: https://github.com/radicle-dev/radicle-upstream/blob/master/CHANGELOG.md
+[rs]: https://raw.githubusercontent.com/radicle-dev/radicle-upstream/master/scripts/reset-state.sh
+[bu]: https://releases.radicle.xyz/radicle-upstream-0.1.5.dmg
```

Comment:
Would this link have to be updated with every release?

Reply:
Correct. Should probably make it 0.X.X.dmg 🤔

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1485/commits/f6a324c2a870a6e7c48f4170d0afe189a5c9291a

Reply:
And to answer your question: the link would have to be updated when we create the release QA issues and paste in the contents of this template.